factorization/configs.py

```diff
@@ -155,7 +158,10 @@ def load_configs(save_ext: str = None) -> list[dict[str, any]]:
 
 
 def load_experimental_results(
-    all_configs: list[dict[str, any]], decorators: list[str] = None, final: bool = False, **kwargs: dict[str, any]
```

Comment:
Can you make sure that you formatter is okay with line that goes all the way to 120 characters?

Reply:
My setting was at 119, bumped it to 120.